CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+# Contribution Guide
+
+There are many ways to be a contributor, and we're here to help you on your way!
+You may:
+
+* Join the discussion and share ideas in our [Slack
+  channel](trustoverip.slack.com) and regular
+  [TSWG](https://wiki.trustoverip.org/display/HOME/Technology+Stack+Working+Group?src=contextnavpagetreemode)
+  and
+  [TATF](https://wiki.trustoverip.org/display/HOME/Technology+Architecture+Task+Force)
+  meetings.
+* Raise an [issue](https://github.com/trustoverip/TechArch/issues) or feature
+  request in our issue tracker
+* Help another contributor with one of their questions, or a documentation
+  review
+* Suggest improvements by supplying a [Pull
+  Request](https://github.com/trustoverip/TechArch/pulls) or opening a
+  [Discussion](https://github.com/trustoverip/TechArch/discussions)
+* Evangelize our work together in conferences, podcasts, and social media
+  spaces.
+
+This guide is for you.
+
+## Table Of Contents
+
+* [Contributing](#contributing)
+  * [Basic Flow](#basic-change-flow)
+  * [Advanced Flow](#advanced-change-flow)
+  * [Commits](#commits)
+  * [Branching](#branching)
+  * [Tagging](#tagging)
+  * [Pull Requests](#pull-requests)
+  * [Review](#review)
+* [Issues](#issues)
+* [Labels](#labels)
+  * [Priority Labels](#priority-labels)
+  * [Type Labels](#type-labels)
+  * [Status Labels](#status-labels)
+
+## Contributing
+
+This section is designed to help you make an edit if you aren't familiar with
+using Github and want to make a change to the TechArch repository.
+
+### Basic Change Flow
+
+If you're not familiar with Github and Git, here's a few simple steps you can
+use to get going and to contribute to the repository *without ever touching the
+command line*.
+
+There is also a [Video Walkthrough](https://www.youtube.com/embed/3CuB0wdJaSU) 
```

Comment:
Note: This is my youtube channel, which I used just for sample here. If ToIP has their own channel, I'm happy to put the video there, or we can remake it. I do think a video needs to be available though!

Reply:
Yes, ToIP has a YouTube channel. @JFleenor has more info on that.